tools/ts-generator/rpc/rpc_gen_js.py

```diff
@@ -228,6 +228,14 @@
         reject(err);
       });
 
+      this.server.on("clientError", (err, socket) => {
```

Comment:
@andresaristizabal is this style of error handling the same as try/catch if you are using typescript?

Reply:
No, `try{}catch(e){}` can get exception into `try` statement and apply `catch(e){}` login based on that error. while `this.server.on("clientError")` is listener for `clientError` event and apply logic that you pass on `(err, socket) =>{}`. if you want  you can use try/catch into login on event listener callback like:

```
this.server.on("clientError", (err, socket) => {
  try{//logic} catch(e) {}
})
```